internal/cli/args.go

```diff
@@ -0,0 +1,80 @@
+package cli
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type argsCmd struct {
+	webPort int
+	clear   bool
+	post    httpPoster
+}
+
+func newArgsCmd() *argsCmd {
+	return &argsCmd{post: http.Post}
+}
+
+func (c *argsCmd) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "args [<tiltfile args>]",
```

Comment:
not a big deal but could consider making the command `tfargs` to be even clearer about what it does/doesn't allow you to set